README.md

```diff
@@ -25,12 +43,30 @@ val activeUser = mutableMapOf<String, Any>(
 val user = Fixture.build<User>(activeUser)
 ```
 
-## Limitations:
+## 🚫 Limitations:
 
 Ktfix does not support the generation of random data every type of data yet,
 checkout our next steps in TODO
 
-### Todo:
-- Add support to generate
-  - Collections  
-- Add support to use of templates
+### 🛠️ Adjustments and improvements:
+
+The project is still under development and future updates will focus on the following tasks:
+- [ ] Generate Date Types
```

Comment:
Some members of this list are already implemented, check the updated version at main branch